views/scan.hbs

```diff
@@ -1,4 +1,38 @@
 {{!< default }}
+
+
+  <div id="banner-info" class="section-container">
+  <div id="banner-sections-wrapper">
+    <div id="banner-left">
+      <div class="banner-content">
+        {{#if foundBreaches }}
+          {{> oh_no }}
+        {{ else }}
+          {{> no_breaches }}
+        {{/if}}
+        </div>
+    </div><!--closes banner-left-->
+    <div id="banner-right">
+      <img id="arrow-divider" src="img/banner_arrow.svg" />
+      <div class="banner-content">
+        {{> sign_up }}
+      </div>
+    </div><!--close banner-right -->
+  </div>
+</div>
+
+{{#if foundBreaches }}
+<div class="content-wrapper">
+  <div class="column-container-twothirds">{{> compromised_accounts }}</div>
+  <div class="column-container-third">{{> what_to_do }}</div>
+{{ else }}
+  {{> tips }}
+{{/if}}
+</div>
```

Comment:
Something feels off here...
I _think_ either the opening `div.content-wrapper` needs to be outside the `{{#if foundBreaches }}`, or this closing `</div>` needs to be within the `{{#if}}`.